src/encoded/static/scss/encoded/_base.scss

```diff
@@ -65,8 +65,23 @@ body { background: $mainThemeColor; font-size: 1rem; }
 
 [hidden] { display: none !important; }
 
-#slot-application, #application {
-	height: 100%;
+#slot-application {
+	height: 100vh;
+}
+
+#application {
+    min-height: 100%;
+
```

Comment:
I don't think that "height: 100vh;" and "min-height: 100%" are necessary. Am I missing a case when they're important?

Reply:
Without either, the footer ends up below the content instead of at the bottom of the window on short pages. Just about every page I saw on this had a different way, many that didn’t work. This way worked on all browsers we support. This is an example of a short page without the min-height:100% enabled.

<img width="1230" alt="no-100" src="https://user-images.githubusercontent.com/1181324/51546167-6f7ca980-1e18-11e9-8433-fbd1da9854d9.png">